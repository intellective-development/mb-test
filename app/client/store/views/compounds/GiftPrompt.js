// @flow
import * as React from 'react';

const GiftPrompt = () => (
  <div className="cm-ae-gift__container">
    <img className="cm-ae-gift__icon" alt="Gift" src="/assets/sprite_sources/icons/gift_icon.png" />
    <span className="cm-ae-gift__text">{'Sending a gift? Use the recipient\'s address.'}</span>
  </div>
);

export default GiftPrompt;
