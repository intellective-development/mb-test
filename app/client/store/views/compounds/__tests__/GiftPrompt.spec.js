
import * as React from 'react';

// import TestProvider from 'client/store/__tests__/utils/TestProvider';
import GiftPrompt from '../GiftPrompt';

describe('GiftPrompt', () => {
  it('renders', () => {
    expect(render(<GiftPrompt />)).toMatchSnapshot();
  });
});
