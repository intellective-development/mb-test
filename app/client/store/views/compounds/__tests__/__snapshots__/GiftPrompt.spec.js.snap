// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiftPrompt renders 1`] = `
<div
  class="cm-ae-gift__container"
>
  <img
    alt="Gift"
    class="cm-ae-gift__icon"
    src="/assets/sprite_sources/icons/gift_icon.png"
  />
  <span
    class="cm-ae-gift__text"
  >
    Sending a gift? Use the recipient's address.
  </span>
</div>
`;
